### ISSUE TITLE
Ability to re-scrape URLs and replace old files

### ISSUE THREAD

**Issue by faucetlol2** [Earlopain/reverser#73]
I ended up making the mistake of scraping an artist's DeviantArt account that had its art as "Watchers-Only Deviations" which resulted in it only downloading blurry preview images.

![image](https://user-images.githubusercontent.com/102884856/210188123-7cc41a98-adb2-4c66-b0eb-1498cc56dbd1.png)

Watching the artist to gain access to the files is easy enough, but there doesn't seem to be any way that I can see to re-scrape the URL and replace the old blurry files with the regular submissions. I thought about just deleting them from the database, but it seems to violate foreign key restraints and I didn't really feel like breaking everything was a good idea.

Not really of great importance to me since I'm not even sure I'll upload any of these files anyway, but could be a nice-to-have. I've experienced artists uploading edited versions or different resolutions _after_ I've scraped their gallery before, but it's rare that happens anyways.

**Comment by Earlopain** [comment]
Makes sense. It would actually be interesting to somehow keep both versions of the image around.

Less interesting for the DeviantArt samples, but more so when the artist uploads an updated version.

**Comment by Earlopain** [comment]
Can you link the artist you've got in that screenshot? Would be nice to have something to properly test with.

**Comment by faucetlol2** [comment]
I believe it was https://www.deviantart.com/zetsuboucchi
